freqtrade/strategy/interface.py

```diff
@@ -610,7 +610,7 @@ def stop_loss_reached(self, current_rate: float, trade: Trade,
         # Initiate stoploss with open_rate. Does nothing if stoploss is already set.
         trade.adjust_stop_loss(trade.open_rate, stop_loss_value, initial=True)
 
-        if self.use_custom_stoploss:
+        if self.use_custom_stoploss and trade.stop_loss < current_rate:
```

Comment:
i don't think this approach works (or the sign is wrong?)

during backtesting - `stop_loss_reached` (this function) is called with `low` as `current_rate` (L528).

as such - this code is preventing the adjustment of the trailing stop whenever the existing stoploss is below low (so below the current candle).
in these cases, stoploss would however not trigger anyway (that would result in a sell below the candle) - so you're preventing raising the stop - but in the wrong case.

While you do see heavily reduced profits in backtesting - the results are not correct, as the stop is simply never moved up UNLESS the prior stop already happens to be within this candle - which somehow seems to be the inverse of what you'd like to achive (reversing the sign will however NOT work)...